khmer/thread_utils.py

```diff
@@ -181,9 +181,9 @@ def do_write(self, outfp):
             except Queue.Empty:
                 continue
 
-            for name, seq, accuracy in g.seqlist:
-                if accuracy:  # write FASTQ; CTB hack.
-                    outfp.write('@%s\n%s\n+\n%s\n' % (name, seq, accuracy))
+            for name, seq, quality in g.seqlist:
+                if quality:  # write FASTQ; CTB hack.
```

Comment:
I can't quite figure out how to pass the individual records to write_record; I'm not sure I understand what the setlist is. I assume a list, yet 

```for item in g.setlist:```
```    write_record(item, outfp)```

doesn't work.